.ci/azure/linux.yml

```diff
@@ -24,11 +24,13 @@ resources:
     type: github
     endpoint: openvinotoolkit
     name: openvinotoolkit/openvino
+    ref: master
```

Comment:
Could it be set using clobal variable? 
env:
     BRANCH_OPENVINO: master
     BRANCH_OPENVINO_CONTRIB: master
etc.